homeassistant/components/androidtv/media_player.py

```diff
@@ -475,6 +477,32 @@ def unique_id(self):
         """Return the device unique id."""
         return self._unique_id
 
+    async def async_get_media_image(self):
+        """Fetch current playing image."""
+        media_data = self.get_raw_media_data()
```

Comment:
This is doing I/O and should not be run from an async method.
```suggestion
        media_data = await self.hass.async_add_executor_job(
            self.get_raw_media_data
        )
```